pycbc/mchirp_area.py

```diff
@@ -67,6 +67,8 @@ def insert_args(parser):
 def from_cli(args):
     mass_limits_sorted = sorted(args.src_class_mass_limits)
     if args.src_class_mass_gap_max:
+        if args.src_class_mass_gap_max < mass_limits_sorted[1]:
+            parser.error('MAX_GAP value cannot be lower than MAX_NS limit')
```

Comment:
Hmm, I don't know if it will work here as you need the parser object, which 'args' by itself does not have access to. You may have to put this check into wherever there is parser.parse_args() instead .. @titodalcanton any advice here?

Reply:
ah, you're right! it should be here then: https://github.com/gwastro/pycbc/blob/master/bin/pycbc_live#L777

Reply:
Or, perhaps cleaner, pass the parser object to `from_cli()` as well i.e. `from_cli(parser, args)`.